docs/source/insights_upload_configuration.rst

```diff
@@ -0,0 +1,39 @@
+Installing the Insights Client
+------------------------------
+The Insights Client requires the insights-core which is why we coupled these repositories together inside of an insights directory. To work with the Insights Client, begin by cloning the repository::
```

Comment:
I would change this section to the following:

To work with the Insights Client, we must also install the Insights Core. To begin, create the insights directory and clone the following repositories::